src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -121,6 +121,24 @@ object WdlExpression {
           case "UnaryNegation" => for(e <- expression) yield e.unaryMinus.get
           case _ => throw new WdlExpressionException(s"Invalid operator: ${a.getName}")
         }
+      case a: Ast if a.getName == "ArrayLiteral" =>
```

Comment:
I suspect the preevaluateExpressionForFilenames will need a case for this too? (i.e. is it possible to have a filename referenced in one of these arrays)? If so, we should also add another case to WdlExpressionToFileSpec.scala